.travis.yml

```diff
@@ -0,0 +1,68 @@
+language: python
+os: linux
+dist: bionic
+
+branches:
+  only:
+    - master
+    - /\d+\.\d+.*/
+  except:
+    - gh-pages
+
+before_install:
+  - pip3 install --upgrade pip
```

Comment:
would be be better do create a venv?

Reply:
Not necessary because we now use `langauge: python`